src/modules/core/components/InfoPopover/MemberInfoPopover/MemberInfoPopover.css

```diff
@@ -2,7 +2,10 @@
 
 .main {
   padding: 20px 25px 25px 25px;
-  width: 470px;
+  width: 480px;
+  letter-spacing: normal;
+  text-align: left;
+  cursor: auto;
```

Comment:
I understand the width update (since that was missed when this was implemented)

But why the reset to default for `letter-spacing`, `text-align` and `cursor` ?

Reply:
PS: these are not ordered properly according to our style lint rules, that's why your build is failing.

If stylelint is not set up in your editor, you can run it manually via `npm run stylelint`

Reply:
There were several occasions where the popover inherited properties of the parent, screwing up the layout, or behavior. The extra props were added to standardize the memberInfoPopover.

Certainly, I need to spend some time improving my editor config. I ran npm linting & type checking manually, so surprised it failed the build.
I want everything automated - manual checking is too easy to forget!!